### ISSUE TITLE
Changing nZEDb Theme from admin menu, doesn't work on complete frontend.

### ISSUE THREAD

**Issue by itsecbern** [nZEDb/nZEDb#2552]
###Expected behaviour
The Theme changes on the login-page as well as in "/browse" & "/movies" -> (complete frontend)

###Actual behaviour
At the moment the theme is just changeing when the user is NOT logget in. -> login-page. after login, there is the old "Alpha theme" is shown again.

###Steps to reproduce the behaviour
Change theme in admin-settings from "Alpha" to an other template -> e.g. "Omicron" (Cache is not the problem!) -> Purged server-cache as well as browser-cache.

**Comment by niel** [comment]
I can't reproduce this using your instructions.

**Comment by itsecbern** [comment]
Okay I try to be more spezific..

The problem I saw, happens by changing the default desing (theme) inside the admin-management page -> (https://URL/admin/site-edit.php?id=) from the "Alpha Template" to someting other.

Whats happening; In my point of view there is some bug how the several templates are included/loaded. Because the new choosed Design (not Alpha) is only applied before the Login happens (Only non logged in users, have the new design. (As soon as they login to nZEDb, the new design is gone & the old Template is loaded))

I hope this is now a better explanation, about the issue.

Kind regards deepwather

**Comment by itsecbern** [comment]
Ok I just validated, my suspicion:

> Test 1:

The link of the NEW choosen template (CSS-file) **before login**: `<link href="/themes/Simple/styles/style.css" rel="stylesheet" media="screen">`

**After Login:** `<link href="/themes/Alpha/styles/style.css" rel="stylesheet" media="screen">`

----

> Test 2, other Template:

**Before Login:**  `<link href="/themes/Omicron/styles/posterwall.css" rel="stylesheet" type="text/css" media="screen"/> <link href="/themes/Omicron/dist/css/AdminLTE.css" rel="stylesheet" type="text/css"/> <link href="/themes/Omicron/dist/css/skins/skin-blue.min.css" rel="stylesheet" type="text/css"/>`

**After Login:**  `<link href="/themes/Alpha/styles/style.css" rel="stylesheet" media="screen">`

----

Conclusion: There must be someting, that overrides or resets the template-variable set by smarty or the CMS it's self.. 

Cheers deepwather

**Comment by Wally73** [comment]
as users can change the theme they are using that one always have the override over the one you pick in admincp

**Comment by niel** [comment]
As Wally says, user specified theme overrides the site wide theme, IF you allow users to choose their own theme.
When I tested I logged in with a user having Charisma as their default theme (or with 'none' specified) and set the site default to Omicron. Everything worked as expected.

BTW If you want to limit your user's choice of themes, remove any directories you do not want them using (renaming them to use a lower case first letter will also work.) However, be aware this will need to be done every time you update unless you commit those changes.
